src/js/groups/querys.ts

```diff
@@ -9,10 +9,35 @@ export const groupQueryKeys = {
     all: () => ["groups"] as const,
     lists: () => ["groups", "list"] as const,
     list: filters => ["groups", "list", ...filters] as const,
+    infiniteLists: () => ["groups", "infiniteList"] as const,
+    infiniteList: (filters: Array<string | number | boolean>) => ["groups", "infiniteList", ...filters] as const,
```

Comment:
```suggestion
    infiniteLists: () => ["groups", "list", "infinite"] as const,
    infiniteList: (filters: Array<string | number | boolean>) => ["groups", "list", "infinite", ...filters] as const,
```
I have been leaning towards this key structure so that you only have to do one invalidation to hit both lists